content/security/authentication.md

```diff
@@ -805,7 +805,14 @@ export class JwtAuthGuard extends AuthGuard('jwt') {
   }
 }
 ```
+#### Extending guards with multiple strategies
 
+In addition to extending the default error handling and authentication logic, we can allow authentication to go through a chain of strategies.
+The first strategy a is successful, redirects or by a mistake will stop the chain. Authentication errors will proceed through each strategy in series, eventually failing if all strategies fail.
```

Comment:
This sentence is quite hard to understand.. What do you mean?

Reply:
see example here  https://github.com/nestjs/docs.nestjs.com/issues/1187